src/common/templates/_ingress.tpl

```diff
@@ -5,15 +5,18 @@ USAGE:
 {{- define "harnesscommon.v1.renderIngress" }}
 {{- $ := .ctx }}
 {{- if $.Values.global.ingress.enabled -}}
+{{- range $idx, $object := $.Values.ingress.objects }}
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ default $.Chart.Name $.Values.nameOverride | trunc 63 | trimSuffix "-" }}
+  name: {{ default $.Chart.Name $.Values.nameOverride | trunc 63 | trimSuffix "-" }}-{{ $idx }}
```

Comment:
Would we want to allow for the setting of the name of the ingress in the values.yaml and set here?

- name: my-ingress-name
  annontations:
 ..etc?

Reply:
Fixed